### PR TITLE
Add all statuscodes from specs

### PR DESCRIPTION
**Pull request by Apollon77**
(some are also needed for GroupsCluster, so I added all)

### CODE REVIEW

File: src/matter/interaction/InteractionMessages.ts [hunk 1]
```diff
@@ -11,6 +11,31 @@ import { AnyT, ArrayT, BooleanT, Field, ObjectT, OptionalField, Template, Unsign
 export const enum StatusCode {
     Success = 0x00,
     Failure = 0x01,
+    InvalidSubscription = 0x7d,
```
Comment (mfucci) [inline src/matter/interaction/InteractionMessages.ts:14]: Can you add a ref to the spec section where does values are defined?

**Review by mfucci — APPROVED** [review]

File: src/matter/interaction/InteractionMessages.ts (hunk 1 above)
Comment (Apollon77) [reply src/matter/interaction/InteractionMessages.ts:14]: done

File: src/matter/interaction/InteractionMessages.ts [hunk 2]
```diff
@@ -7,10 +7,35 @@
 import { TlvType } from "../../codec/TlvCodec";
 import { AnyT, ArrayT, BooleanT, Field, ObjectT, OptionalField, Template, UnsignedIntT, UnsignedLongT } from "../../codec/TlvObjectCodec";
 
-// See project-chip src/protocols/interaction_model/StatusCodeList.h
+// From [Matter Core Specification R1.0], section 8.10
```
Comment (mfucci) [inline src/matter/interaction/InteractionMessages.ts:10]: Maybe use a real jsDoc comment

**Review by mfucci — APPROVED** [review]

**Review by mfucci — APPROVED** [review]

**Comment by Apollon77** [comment]
@mfucci one last mini change :-)

**Comment by Apollon77** [comment]
feel free to merge after appoval

**Review by mfucci — APPROVED** [review]

**Pull request merged**